### DISCUSSION TITLE
【润的哲学思辨】润是背叛民族吗：关于民族性与阶级性的探讨

### DISCUSSION THREAD

**Discussion by Yilun-D** [The-Run-Philosophy-Organization/run#403] [General]
Deleted

**Comment by AntEffect** [comment]
参考马逆的看法，国家民族政府只不过都是寄生在普通人身上的“病原体”而已。

**Comment by walker1996** [comment]
润并不是背叛民族的做法。但可以确定的是，把润的做法和民族主义和意识形态（强绑定）是不妥当的。当然，一个人选择润的理由可以有上述两种原因，这是一个人的自由，也是我理解的自由。
但像楼上一刀切把一个润的行为上升到意识形态和民族主义问题上，而枉顾具体的现实情况和实践的二极管思维，我是强烈反对的。
楼上那种说辞客观上也是民族主义的观点。只不过他是反民族主义政府的民族主义罢了。
